www/md_base/src/icons/gear-outline.svg

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Good idea.

On Fri, Apr 10, 2015 at 8:26 PM Pierre Tardy <notifications@github.com>
wrote:

> In www/md_base/src/icons/gear-outline.svg
> <https://github.com/buildbot/buildbot/pull/1626#discussion_r28140177>:
>
> > @@ -0,0 +1,40 @@
> > +<?xml version="1.0" encoding="utf-8"?>
>
> Ok, I think I understand more. Its okay if the svg are concatenated.
>
> What should be avoided is to include icons directly in git, we should
> rather use bower to fetch those icons in bower package.
>
> This will need update of guanlecoja to support configuration of which
> icons we want in the bower section:
>
> "material-design-icons":
> version: "1.0.1"
> files: [ "notification/svg/production/ic_system_update_48px.svg",
> "notification/svg/production/ic_more_48px.svg" ]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/buildbot/buildbot/pull/1626/files#r28140177>.
>